client/landing/stepper/declarative-flow/internals/steps-repository/free-setup/styles.scss

```diff
@@ -0,0 +1,16 @@
+@import "../style";
+
+$font-family: "SF Pro Text", $sans;
+
+.free-setup {
+	.step-container__content {
+		display: flex;
+		justify-content: center;
+		margin-bottom: 30px;
+		min-height: unset;
+	}
+
+	.free-setup .step-container__header {
```

Comment:
I had just mimicked the other setup forms. Oddly we do this for the link in bio and newsletter setup forms, too. And looking at the markup, it seems we actually do repeat .newsletter-setup / .link-in-bio-setup / .free-setup class on both the parent and on the step-container here. So both `.free-setup .step-container__header` and `.free-setup .free-setup .setup-container` are both valid. I'm hesitant to remove the extra class from markup in case it's used for other things, but clearly the extra selector is not needed here, so I've removed it.